demo/server/src/modules/index.ts

```diff
@@ -4,10 +4,13 @@ import * as express from 'express';
 import { connect as syncConnector } from './datasync/Connector';
 import { router as syncRouter } from './datasync/Router';
 import { init as authInit } from './passport-auth';
+import { init as initKeycloak } from './keycloak';
 
 export let securityMiddleware: EndpointSecurity;
 export let logger: Logger;
 
+const useKeycloak = true;
```

Comment:
This was mainly for testing purposes: I think the existence of a `keycloak.json` (with parseable JSON) in the server route would be a good check on whether to use Keycloak or not.